src/mantarray_waveform_analysis/__init__.py

```diff
@@ -25,7 +25,7 @@
 from .transforms import noise_cancellation
 
 if (
-    6 < 9
+    5 < 10
```

Comment:
was this just for fun?

Reply:
For some reason pylint was flagging it as a duplicate code section, not sure why since I'm pretty sure we have pylint ignore 'duplicate code' when it pertains to imports

Reply:
ah ok. I think the `if` statement (correctly IMO) tells pylint that this is not an "import" section

Reply:
I see